addons/css-fix/userstyle.css

```diff
@@ -0,0 +1,18 @@
+
+.blocklyWidgetDiv .goog-menu {
+overflow: hidden; /* Hide scrollbars */
+}
+.sprite-selector_items-wrapper_4bcOj.box_box_2jjDp {
+overflow: hidden;
+}
+.sprite-selector_items-wrapper_17MEf {
+overflow: hidden; /* Hide scrollbars */
+}
+.s3devSTT {
+border-top: 1px solid hsla(0, 0%, 0%, 0.15);
+padding: 8px 12px;
+}
```

Comment:
Looks like this is styling an element added by the developer tools addon? If so, you should change that addon, not add a new one that changes it.

Reply:
Sorry, that code is supposed to be something else, I will fix it as soon as possible